src/components/YoutubeCard/YoutubeCard.component.jsx

```diff
@@ -7,14 +7,20 @@ import {
   CardDescription,
 } from './YoutubeCard.styles';
 
-function Card(video) {
-  const { videoInfo } = video;
+function Card(props) {
+  const { videoInfo, onVideoClick } = props;
+  const { thumbnails, title, description } = videoInfo.snippet;
```

Comment:
I would change this to have these props passed directly

```suggestion
const { thumbnails, title, description, onVideoClick } = props;
```